.github/workflows/main.yaml

```diff
@@ -0,0 +1,38 @@
+name: Build and Deploy to ECR
+on:
+  push:
+    branches: [ "develop" ]
+jobs:
+  build-and-push-ecr:
+    runs-on: ubuntu-latest
+    environment: development
+    env:
+      ARKCASE_VERSION: 2021.03-RC11
+    steps:
+      - uses: actions/checkout@v2
+      - name: configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+      - name: download artifacts
+        run: ./getArtifact.sh
+        env:
+          ARKCASE_VERSION: 2021.03-RC11
+      - name: login to ECR repo
+        uses: aws-actions/amazon-ecr-login@v1
+        id: login-ecr
+      - name: build and publish to ecr
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: arkcase_core
+          TOMCAT_VERSION: 9.0.50 
+          TOMCAT_MAJOR_VERSION: 9
+          SYMMETRIC_KEY: 99999999
+          MARIADB_CONNECTOR_VERSION: 2.2.5
+          RESOURCE_PATH: artifacts
+        run: |
+          docker build -t  $ECR_REGISTRY/$ECR_REPOSITORY:$ARKCASE_VERSION --build-arg TOMCAT_VERSION --build-arg  TOMCAT_MAJOR_VERSION --build-arg  SYMMETRIC_KEY --build-arg MARIADB_CONNECTOR_VERSION --build-arg RESOURCE_PATH .
```

Comment:
I see the `Dockerfile` still has equivalent `ARG` elements.  Do these environment variables and build args take precedence over the ARGs in the Dockerfile?

Reply:
Yes @david-oc-miller

Reply:
OK thanks, I see how it works from here: https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg.

I do notice that you are not passing ARKCASE_VERSION as a build-arg?

Reply:
Thanks for the notice. Added ARKCASE_VERSION now

Reply:
@naveen-armedia I still don't see where you are passing `ARKCASE_VERSION` in a `build-arg`.  Do you need to push another commit?